resources/sdk/pureclouddotnet/templates/Configuration.mustache

```diff
@@ -62,6 +66,47 @@ namespace {{packageName}}.Client
             Timeout = timeout;
             ShouldRefreshAccessToken = shouldRefreshAccessToken;
             RefreshTokenWaitTime = refreshTokenWaitTime;
+
+            if (logger != null)
+                Logger = logger;
+            else
+                Logger = new Logger();
+
+            if (String.IsNullOrEmpty(configFilePath))
+            {
+                string homePath = (Environment.OSVersion.Platform == PlatformID.Unix || 
+                                Environment.OSVersion.Platform == PlatformID.MacOSX)
+                    ? Environment.GetEnvironmentVariable("HOME")
+                    : Environment.ExpandEnvironmentVariables("%HOMEDRIVE%%HOMEPATH%");
+                ConfigFilePath = Path.Join(homePath, "genesysclouddotnet", "config");
+            }
+            else
+            {
+                ConfigFilePath = configFilePath;
+            }
+
+            // applyConfigFromFile();
+            if (AutoReloadConfig)
+            {
+                ThreadStart childref = new ThreadStart(RunConfigChecker);
+                Console.WriteLine("In Main: Creating the Child thread");
+                Thread childThread = new Thread(childref);
+                childThread.Start();
+            }
+        }
+
+        private void applyConfigFromFile()
```

Comment:
Why is this function empty?

Reply:
I think you're viewing the version from the first commit. If you look at the changes from https://github.com/purecloudlabs/platform-client-sdk-common/pull/153/files it should be all up-to-date